pkg/executor/bacalhau/bacalhau.go

```diff
@@ -208,7 +208,7 @@ func (executor *BacalhauExecutor) getJobID(
 	outputError := strings.Join(strings.Fields(strings.Join(splitOutputs[1:], " ")), " ")
 
 	if outputError != "" {
-		return "", fmt.Errorf("error running command %s -> %s, %s", deal.ID, outputError, runOutput)
+		log.Error().Msgf("error parsing output %s -> %s, %s", deal.ID, outputError, runOutput)
```

Comment:
```suggestion
		log.Error().Msgf("error found while parsing output %s -> %s, %s", deal.ID, outputError, runOutput)
```
Possible edit. It's not a parsing error, but an error we found while parsing.

Reply:
Do you have thoughts about it being `ERR` vs `WARN`?

Reply:
Tough call. On the one hand, the possible error we know about (around the version check) is more of a warning. On the other hand, could there be other errors that aren't a warning?

I'd say let's go with the common case we know about and use `WARN`. If we start seeing other errors, we can revisit, but this may not matter once we have upgraded to the latest Bacalhau.

Reply:
yeah, once we get fully over to the bacalhau http api, a lot of this code become moot. 👍